### ISSUE TITLE
Pass options to proxy middleware

### ISSUE THREAD

**Issue by adailey14** [halt-hammerzeit/web-service#2]
It would be useful if I could pass options into proxy middleware, like this:

```js
web.proxy('/api', 'http://myserver.com/api', {
  changeOrigin: true,
  autoRewrite: true,
});
```

**Comment by halt-hammerzeit** [comment]
Do you think these settings should be default?

**Comment by adailey14** [comment]
Probably not, but I'm not sure. I need to use those options to make heroku recognize which app to route requests to. Heroku routes based on the origin host header I believe. For non-heroku type of environment these options are probably not needed.

**Comment by halt-hammerzeit** [comment]
Ok then, I released `web-service@0.1.20`, and the third parameter is the options object.
